src/libs/ajax/workflows-app/Cbas.js

```diff
@@ -19,6 +23,14 @@ export const Cbas = (signal) => ({
       const res = await fetchFromProxy(cbasUrlRoot)('api/batch/v1/methods', _.mergeAll([authOpts(), jsonBody(payload), { signal, method: 'POST' }]));
       return res.json();
     },
+    patch: async (cbasUrlRoot, methodId, payload) => {
```

Comment:
I went back and forth on this and landed here for no particular reason... I'll take your suggestion, but also change this from `patch` to `archive` in that case

Reply:
actually, I think if we want to align this function with what the caller is capable of doing, it shouldn't have a boolean argument at all, because we don't support un-archiving right now